examples/2_intermediate/mta.ipynb

```diff
@@ -0,0 +1,495 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f4d848d1-dd58-4b75-9662-377cb66b5be9",
+   "metadata": {},
+   "source": [
+    "# Using CSP to analyze MTA data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6906efcb-324a-4bc1-b910-6d8905c0adb0",
+   "metadata": {},
+   "source": [
+    "The NYC Metropolitan Transportation Authority provides [an API for developers](https://api.mta.info), and we'll explore the MTA's realtime [GTFS-rt](https://developers.google.com/transit/gtfs-realtime) transportation data feed."
```

Comment:
Let's add a link to https://new.mta.info/developers as well since this contains the Reference files for the subway feed, as well as links to their other feeds (accessibility data for example).

Reply:
This link unfortunately does not work for me - I get
```
Access Denied
You don't have permission to access "http://new.mta.info/developers" on this server.

Reference #18.23c41002.1715115812.c935d9f4

https://errors.edgesuite.net/18.23c41002.1715115812.c935d9f4"
```

I have tried asking the MTA developer portal for help and they seemed to imply this is a temporary problem but I am suspecting this is maybe geoblocked?

Reply:
Wow that's strange, I just clicked it and it worked? It should be an `https` in the URL but that shouldn't matter.

Reply:
Yeah wow, I see it being geoblocked in at least Hong Kong, Germany, Spain, Italy, Russia, accessible from US, UK, Sweden, Japan, Singapore.

Reply:
I can access the data though, and got to use it from inspecting the feed, so I don't think this is a blocker - I can add a note that the documentation for the API may be hard to access in certain regions.